sympy/stats/stochastic_process_types.py

```diff
@@ -658,6 +680,37 @@ def probability(self, condition, given_condition=None, evaluate=True, **kwargs):
         raise NotImplementedError("Mechanism for handling (%s, %s) queries hasn't been "
                                 "implemented yet."%(condition, given_condition))
 
+    def symbolic_probability(self, condition, new_given_condition, rv, min_key_rv):
```

Comment:
Shouldn't it be private? I mean `_symbolic_probability`.

Reply:
I'll change that